scripts/install-binary.js

```diff
@@ -28,23 +29,26 @@ if (jqExists()) {
   process.exit(0)
 }
 
-const build = new BinBuild()
-  .src(`${JQ_INFO.url}/${JQ_INFO.version}/${JQ_INFO.version}.tar.gz`)
+if (process.platform === 'win32' || process.platform === 'win64') {
```

Comment:
Are you sure around that? I never touch cross-platform scripts on Node, but seems incorrect.
Could you please add some info there or try to use some tested lib to do that?